ops/scripts/tf_init.sh

```
@@ -134,8 +141,8 @@ encoded_gitlab_project=$(echo $gitlab_project | sed -e 's/\//%2F/g')
 
 
 # Ensure we are in the environment-specific directory
-if [ "envs/$target_environment" != `pwd | rev | cut -d"/" -f 1,2 | rev` ];then
-  echo "You are not in the correct directory given the specified parameters. you should be in 'envs/$target_environment'"
+if [ "$target_environment" != `pwd | rev | cut -d"/" -f 1 | rev` ];then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)